examples/hubble/hubble-network-policy.yaml

```diff
@@ -0,0 +1,66 @@
+---
+apiVersion: cilium.io/v2
+kind: CiliumNetworkPolicy
+metadata:
+  name: hubble-relay
+  namespace: kube-system
+spec:
+  endpointSelector:
+    matchLabels:
+      k8s-app: hubble-relay
+  egress:
+    - toEntities:
+        - host
+        - remote-node
+      toPorts:
+        - ports:
+            - port: "4244"
+              protocol: TCP
+    - toEndpoints:
+        - matchLabels:
+            k8s-app: kube-dns
```

Comment:
Please also match the namespace here: https://docs.cilium.io/en/stable/security/policy/kubernetes/#allow-all-cilium-managed-endpoints-to-communicate-with-kube-dns